src/routes.js

```diff
@@ -1,6 +1,5 @@
 import HelpPage from './pages/help/help';
 import LandingPage from './pages/landing/landingPage';
-// import StaticLandingPage from './pages/staticLanding/staticLandingPage';
```

Comment:
Should the 'staticLandingPage.js' file be removed as well, unless it's used elsewhere